pkg/kubernetes/apireference.go

```diff
@@ -0,0 +1,77 @@
+package kubernetes
+
+import (
+	"strings"
+
+	openapi_v2 "github.com/google/gnostic/openapiv2"
+)
+
+func (k *K8sApiReference) GetApiDocV2(field string) string {
+	startPoint := ""
+	// the path must be formated like "path1.path2.path3"
+	paths := strings.Split(field, ".")
+	group := strings.Split(k.ApiVersion.Group, ".")
+
+	// openapiSchema := openapi_v2.Document{}
+
+	// openapiSchema, err := k.Discovery.OpenAPISchema()
+	// if err != nil {
+	// 	return ""
+	// }
+
+	definitions := k.OpenapiSchema.GetDefinitions().GetAdditionalProperties()
+
+	// extract the startpoint by searching the highest leaf corresponding to the requested group qnd kind
+	for _, prop := range definitions {
+		if strings.HasSuffix(prop.GetName(), group[0]+"."+k.ApiVersion.Version+"."+k.Kind) {
```

Comment:
Can we use `fmt.Sprintf` to be more consistent ?

Reply:
fixed